src/Breadcrumb.php

```diff
@@ -60,9 +60,16 @@ final class Breadcrumb implements \JsonSerializable
 
     /**
      * This constant defines the critical level for a breadcrumb.
+     *
+     * @deprecated 2.2.2 Use fatal instead. Will be removed in a later version.
```

Comment:
Please change this description to `@deprecated since version 2.2.2, to be removed in 3.0` so that it's consistent with the rest of the code